grammars/html (ruby - erb).cson

```diff
@@ -71,8 +71,8 @@
         'beginCaptures':
           '0':
             'name': 'punctuation.section.embedded.begin.erb'
-        'contentName': 'source.ruby'
-        'end': '(-?%)>'
+        'contentName': 'source.ruby.embedded'
```

Comment:
To make this consistent with other languages, I'd suggest using `source.ruby.embedded.erb` instead.  Ditto below.